### ISSUE TITLE
Installation issues (pip, missing 'hg')

### ISSUE THREAD

**Issue by thereallinusrg** [SALib/SALib#289]
I have issues trying to install SALib on Arch via pip.
Could anyone hint me in a direction to solve this?

```bash
ERROR: Command errored out with exit status 1:
     command: /usr/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-_iecc01h/SALib/setup.py'"'"'; __file__='"'"'/tmp/pip-install-_iecc01h/SALib/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-_iecc01h/SALib/pip-egg-info
         cwd: /tmp/pip-install-_iecc01h/SALib/
    Complete output (9 lines):
    running egg_info
    creating /tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info
    writing /tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info/PKG-INFO
    writing dependency_links to /tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info/dependency_links.txt
    writing entry points to /tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info/entry_points.txt
    writing requirements to /tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info/requires.txt
    writing top-level names to /tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info/top_level.txt
    writing manifest file '/tmp/pip-install-_iecc01h/SALib/pip-egg-info/SALib.egg-info/SOURCES.txt'
    error: [Errno 2] No such file or directory: 'hg'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by ConnectedSystems** [comment]
Hi @thereallinusrg 

I'm not much of a linux person, much less Arch, but I'll try to help.
Could you post up the output from `python setup.py egg_info` after you attempt an install?

**Comment by thereallinusrg** [comment]
After cloning the repo and runing `python setup.py egg_info `  I got this:

`raise LookupError(
LookupError: setuptools-scm was unable to detect version for '/home/linus/Downloads/SALib-master'.
Make sure you're either building from a fully intact git repository or PyPI tarballs. Most other sources (such as GitHub's tarballs, a git checkout without the .git folder) don't contain the necessary metadata and will not work.
For example, if you're using pip, instead of https://github.com/user/proj/archive/master.zip use git+https://github.com/user/proj.git#egg=proj`

So I did:
`pip install git+https://github.com/SALib/SALib.git#egg=SALib`
Which was successful, thank you!
